app/furniture/marketplace/orders_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Marketplace
   class OrdersController < Controller
-    expose :order, scope: -> { orders }, model: Order
+    expose :order, scope: -> { orders.includes(:events, ordered_products: [:product]) }, model: Order
```

Comment:
Lilliputian Zee would do `events` independently as part of this PR, and then the `ordered_products` after; but Gilligan Zee is just stomp stomp stomping on the orders